wp-cdn-rewrite.php

```diff
@@ -122,6 +122,12 @@ public function filter_urls($content) {
 				$xml_end = '<';
 			}
 			$extensions = join('|', array_filter(array($this->file_extensions, $this->css_file_extensions, $this->js_file_extensions)));
+
+			// replace srcset values
+			$srcset_regex = '#<img[^\>]*[^\>\S]+srcset=[\'"]('.quotemeta($root_url).'(?:([^"\'\s,]+)('.$extensions.')\s*(?:\s+\d+[wx])(?:,\s*)?)+)["\'][^>]*?>#';
```

Comment:
@smccafferty you should just be able to check against $this->file_extensions within srcset, right?